corpusreaders/src/test/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/mascReaderTests/MascXCESReaderTest.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * This software is released under the University of Illinois/Research and Academic Use License. See
+ * the LICENSE file in the root folder for details. Copyright (c) 2016
+ *
+ * Developed by: The Cognitive Computation Group University of Illinois at Urbana-Champaign
+ * http://cogcomp.cs.illinois.edu/
+ */
+package edu.illinois.cs.cogcomp.nlp.corpusreaders.mascReaderTests;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+import edu.illinois.cs.cogcomp.nlp.corpusreaders.mascReader.MascXCESReader;
+
+public class MascXCESReaderTest {
+
+    @Test
+    public void testCreateTextAnnotation() throws Exception {
+        Logger.getLogger(MascXCESReader.class).setLevel(Level.INFO);
+
```

Comment:
please add a note somewhere -- could be this class's javadoc -- that indicates to the user that they will need to change the hard-coded path to run on non-cogcomp servers.